neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/view/activity/VariantListActivity.kt

```diff
@@ -85,6 +88,21 @@ class VariantListActivity : AppCompatActivity(), VariantListPresenter.ViewSurfac
         navigate(EditVariantActivity::class, EditVariantPresenter.ViewSurface.EXTRA_NAME to name)
     }
 
+    override fun goToMainApplication() {
+        val resolvedInfos = packageManager.queryIntentActivities(Intent(Intent.ACTION_MAIN), PackageManager.GET_RESOLVED_FILTER)
```

Comment:
@AlgoRyan yes it would. I will optimise this algorithm a little so that it will launch the default main intent if its not neanderthal, or the first for that application if it is